src/devices/Common/Iot/Device/Graphics/BdfFont.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Graphics
+{
+    public class BdfFont
+    {
+        public int Width  { private set; get; }
+        public int Height { private set; get; }
+        public int XDisplacement { private set; get; }
+        public int YDisplacement { private set; get; }
+        public int DefaultChar  { private set; get; }
+        public int CharsCount   { private set; get; }
+
+        private Dictionary<int, int> GlyphMapper { set; get; }
```

Comment:
Some tiny comment here explaining what are you mapping to map would be useful

Reply:
I'll do that.